gimme

```diff
@@ -153,7 +158,14 @@ _checkout() {
 # _extract "file.tar.gz" "dir"
 _extract() {
 	mkdir -p "${2}"
-	tar -xf "${1}" -C "${2}" --strip-components 1
+
+	if [[ "${1}" == *.tar.gz ]] ; then
+		tar -xf "${1}" -C "${2}" --strip-components 1
+	else
+		unzip -q "${1}" -d "${2}"
+		mv "${2}"/go/* "${2}"
```

Comment:
Won't this potentially lose "hidden files"?  Any ideas for solving that without causing too much trouble?